src/compute-client/src/controller/instance.rs

```diff
@@ -245,18 +245,24 @@ where
             BTreeSet::new()
         };
 
-        // Initialize frontier tracking the new replica.
+        // Initialize frontier tracking the new replica
```

Comment:
```suggestion
        // Initialize frontier tracking for the new replica
```

I know this was actually my own typo :D